components/hdw-oled/ssd1306.c

```diff
@@ -379,6 +379,8 @@ uint8_t priorFb[(OLED_WIDTH * (OLED_HEIGHT / 8))] = {0};
 
 bool fbChanges = false;
 
+display_t oledDisp;
```

Comment:
A part of me wanted to have these functions be stateless and have everything in `display_t`, but that ended up being a little too idealistic for this project.

Reply:
Also this should be a pointer.

Reply:
your wish is my command